frontend/src/utils/crawler.ts

```diff
@@ -29,11 +29,11 @@ export const DEPTH_SUPPORTED_SCOPES = [
   "any",
 ];
 
-export function isActive({ state, stopping }: Partial<Crawl | QARun>) {
+export function isActive({ state }: Partial<Crawl | QARun>): boolean {
   return (
     (state &&
       (activeCrawlStates as readonly string[]).includes(state as string)) ||
-    stopping === true
+    false
   );
 }
```

Comment:
This could be simplified a little:
```ts
export function isActive({ state }: Partial<Crawl | QARun>) {
  return (activeCrawlStates as readonly (typeof state)[]).includes(state);
}

```

`Array.prototype.includes` can be passed `undefined` as an argument, and will correctly return `false` if the array doesn't contain an `undefined`, which it doesn't in this case.

Reply:
Thanks, updated!